app/views/sessions/_filters.html.erb

```diff
@@ -1,27 +1,27 @@
 <%# locals: (resource:) %>
 
-<div class="flex justify-between w-full mb-6">
+<div class="flex justify-between w-full mb-4">
   <div class="flex items-center gap-2 overflow-x-scroll no-scrollbar">
-    <%= link_to(
-      'All dates',
-      url_for([resource, session_filter_params.to_h]),
-      class: [
-        "flex min-w-fit items-center justify-center px-4 py-2 rounded-full font-medium",
-        "bg-red text-white": params[:starts_at].blank?,
-        "bg-white text-red hover:bg-[#FAE7E8] focus:bg-[#FAE7E8]": params[:starts_at].present?
-      ]
-    ) %>
     <% current_conference.session_dates.each do |session_date| %>
       <%= link_to(
         session_date.strftime("%a, %d"),
-        url_for([resource, session_filter_params.to_h.merge(starts_at: session_date)]),
+        url_for([resource, session_filter_params.to_h.merge(starts_at: session_date)]) + current_day_anchor(session_date),
```

Comment:
url_for uses `anchor` key as an actual param, not anchor element contrary as link_to, that's why we have to append the anchor like this in this case